src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceEventRepository.cs

```diff
@@ -0,0 +1,159 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Documents.Linq;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Repository
+{
+    /// <summary>
+    /// Handles instanceEvent repository
+    /// </summary>
+    public class InstanceEventRepository : IInstanceEventRepository
+    {
+        private readonly Uri _databaseUri;
+        private readonly Uri _collectionUri;
+        private readonly string databaseId;
+        private readonly string collectionId = "instanceEvent";
+        private readonly string partitionKey = "/instanceId";
+        private static DocumentClient _client;
+        private readonly AzureCosmosSettings _cosmosettings;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InstanceEventRepository"/> class
+        /// </summary>
+        /// <param name="cosmosettings">the configuration settings for cosmos database</param>
+        public InstanceEventRepository(IOptions<AzureCosmosSettings> cosmosettings)
+        {
+            // Retrieve configuration values from appsettings.json
+            _cosmosettings = cosmosettings.Value;
+            databaseId = _cosmosettings.Database;
+
+            ConnectionPolicy connectionPolicy = new ConnectionPolicy
+            {
+                ConnectionMode = ConnectionMode.Gateway,
+                ConnectionProtocol = Protocol.Https,
+            };
+
+            _client = new DocumentClient(new Uri(_cosmosettings.EndpointUri), _cosmosettings.PrimaryKey, connectionPolicy);
+
+            _databaseUri = UriFactory.CreateDatabaseUri(_cosmosettings.Database);
+            _collectionUri = UriFactory.CreateDocumentCollectionUri(_cosmosettings.Database, collectionId);
+
+            _client.CreateDatabaseIfNotExistsAsync(new Database { Id = databaseId }).GetAwaiter().GetResult();
+
+            DocumentCollection documentCollection = new DocumentCollection { Id = collectionId };
+            documentCollection.PartitionKey.Paths.Add(partitionKey);
+
+            _client.CreateDocumentCollectionIfNotExistsAsync(
+                _databaseUri,
+                documentCollection).GetAwaiter().GetResult();
+
+            _client.OpenAsync();
+        }
+
+        /// <summary>
+        ///  Inserts new instance event into the instanceEvent collection.
+        /// </summary>
+        /// <param name="item">Instance event to be stored./param>
```

Comment:
All methods from the interface can use <inheritdoc /> so you don't have to write the same code comments twice. Reminder for next time, don't have to fix it for this file unless you want to

Reply:
Changed i to /// <inheritdoc/> for all functions.